pkg/registry/deployment/etcd/etcd.go

```diff
@@ -62,7 +62,7 @@ func NewREST(s storage.Interface, storageDecorator generic.StorageDecorator) (*R
 
 	newListFunc := func() runtime.Object { return &extensions.DeploymentList{} }
 	storageInterface := storageDecorator(
-		s, 100, &extensions.Deployment{}, prefix, false, newListFunc)
+		s, 100, &extensions.Deployment{}, prefix, true, newListFunc)
```

Comment:
Please use `deployment.NamespaceScoped()` so that future copy-paste cycles won't make the same error. (same in all other places.